.github/workflows/build.yml

```diff
@@ -0,0 +1,50 @@
+name: Build CI
+
+on: [pull_request, push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Dump GitHub context
+      env:
+        GITHUB_CONTEXT: ${{ toJson(github) }}
+      run: echo "$GITHUB_CONTEXT"
+    - name: Translate Repo Name For Build Tools filename_prefix
+      id: repo-name
+      run: |
+        echo ::set-output name=repo-name::$(
+        echo ${{ github.repository }} |
+        awk -F '\/' '{ print tolower($2) }' |
+        tr '_' '-'
+        )
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
+    - name: Versions
+      run: |
+        python3 --version
+    - name: Checkout Current Repo
+      uses: actions/checkout@v1
+      with:
+        submodules: true
+    - name: Checkout tools repo
+      uses: actions/checkout@v2
+      with:
+        repository: adafruit/actions-ci-circuitpython-libs
+        path: actions-ci
+    - name: Install deps
+      run: |
+        source actions-ci/install.sh
+    - name: Library version
+      run: git describe --dirty --always --tags
+    - name: PyLint
+      run: |
+        pylint $( find . -path './adafruit*.py' -o -maxdepth 1 -path '*.py' ! -name 'setup.py' )
```

Comment:
Took me a minute, but I finally understood that you were trying to handle the files that don't match `adafruit_*.py`. But, when I run this, I get the following. While it works for SimpleIO, it doesn't work for AdafruitIO.
```
~/Dev/circuitpython_libs/Adafruit_CircuitPython_AdafruitIO:$> find . -path './adafruit*.py' -o -maxdepth  1 -path '*.py' ! -name 'setup.py'
find: warning: you have specified the -maxdepth option after a non-option argument -path, but options are not positional (-maxdepth affects tests specified before it as well as those specified after it).  Please specify options before other arguments.

~/Dev/circuitpython_libs/Adafruit_CircuitPython_AdafruitIO:$> cd ../Adafruit_CircuitPython_SimpleIO
~/Dev/circuitpython_libs/Adafruit_CircuitPython_SimpleIO:$> find . -path './adafruit*.py' -o -maxdepth  1 -path '*.py' ! -name 'setup.py'
find: warning: you have specified the -maxdepth option after a non-option argument -path, but options are not positional (-maxdepth affects tests specified before it as well as those specified after it).  Please specify options before other arguments.

./simpleio.py
~/Dev/circuitpython_libs/Adafruit_CircuitPython_SimpleIO:$> 
```

I'd assume the cause is the `-maxdepth` argument, since it applies to both conditions, not just the latter.

Outside of that, I also wouldn't put this in the cookiecutter. This would basically _allow_ future libraries to use a non-standard naming convention. But that's my opinion. :smile: